sonar-db/src/main/java/org/sonar/db/version/v564/CleanUsurperRootComponents.java

```diff
@@ -40,15 +43,15 @@ public void execute(Context context) throws SQLException {
     cleanSnapshotWithIncorrectRoot(context);
   }
 
-  private static void fixSnapshotScopeAndQualifier(Context context) throws SQLException {
+  private void fixSnapshotScopeAndQualifier(Context context) throws SQLException {
     MassUpdate massUpdate = context.prepareMassUpdate();
-    massUpdate.select("select" +
+    massUpdate.select(fixSqlConditions(getDb().getDialect(), "select" +
       " sn.id,p.scope,p.qualifier" +
       " from" +
       " snapshots sn, projects p" +
       " where" +
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal " where" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)